tableaudocumentapi/connection.py

```diff
@@ -26,6 +26,11 @@ def __init__(self, connxml):
         self._dbname = connxml.get('dbname')
         self._server = connxml.get('server')
         self._username = connxml.get('username')
+        self._authentication = connxml.get('authentication')
+        self._class = connxml.get('class')
+
+    def __repr__(self):
```

Comment:
feels weird to me to not use the existing representation but I have not worked through all the debug output yet so am not yet "passionate". Note, for xml, the element names start lower case. :) Thanks!!